ucSubmitNews.ascx.vb

```diff
@@ -336,6 +342,38 @@ Namespace Ventrian.NewsArticles
 
         End Sub
 
+        Private Sub BindTags()
+            If (ArticleSettings.UseStaticTagsListSubmit) Then
+                txtTags.Visible = False
+                lblTags.Visible = False
+
+                Dim objTagController As New TagController
+                Dim objTags As ArrayList = objTagController.List(ModuleId, Null.NullInteger)
+
+                objTags.Sort()
+                lstTags.DataSource = objTags
+                lstTags.DataBind()
+            Else
+                lstTags.Visible = False
+            End If
+        End Sub
+
+        Private Sub SelectTags(ByVal tagList As String)
+            Dim objTagController As New TagController
+            For Each tag As String In tagList.Split(","c)
+                If (tag <> "") Then
+                    Dim objTag As TagInfo = objTagController.Get(ModuleId, tag)
+
+                    If Not (objTag Is Nothing) Then

```

Comment:
Consider using [the `IsNot` operator](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/operators/isnot-operator)
```suggestion
                    If objTag IsNot Nothing Then
```